plugins/entity-validation/src/components/EntityValidationPage/EntityValidationPage.tsx

```diff
@@ -66,15 +66,15 @@ export const EntityValidationPage = (props: {
         subtitle="Tool to validate catalog-info.yaml files"
       />
       <Content>
-        <Grid container direction="row">
+        <Grid container>
           <Grid item md={9} xs={12}>
             <TextField
-              style={{ width: '100%' }}
+              fullWidth
               label="File Location"
               margin="normal"
               variant="outlined"
               required
-              value={defaultLocation}
+              value={locationUrl}
```

Comment:
The idea behind setting the defaultLocation as a value was to have something filled in. However, I made a mistake when changing this.

I would do it this way:

* Set the state of `locationUrl` with the value of `defaultLocation` instead of an empty string
* In the TextField, set `defaultValue={defaultLocation}` and `value={locationUrl}`. This fixes the issue

Reply:
This was what I had originally, but wasn't sure as to the desired functionality. I can make that change.

Reply:
I agree with @ivangonzalezacuna; this aligns to what I was going to do when I got around to making the PR. Thanks @webark for beating me to it